openstack/clustering/v1/policies/requests.go

```diff
@@ -54,3 +58,63 @@ func List(client *gophercloud.ServiceClient, opts ListOptsBuilder) pagination.Pa
 		return p
 	})
 }
+
+// CreateOpts params
+type CreateOpts struct {
+	Name string `json:"name"`
+	Spec Spec   `json:"spec"`
+}
+
+type Spec struct {
+	Description string                 `json:"description"`
+	Properties  map[string]interface{} `json:"properties"`
+	Type        string                 `json:"type"`
+	Version     Version                `json:"version"`
```

Comment:
Let's change the type back to `string`.

Reply:
I couldn't get it to work with `Version` set to type `string` so I ended up changing it to `interface{}`.  See my comments below.